tron/actioncommand.py

```diff
@@ -203,6 +207,14 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    @staticmethod
+    def from_json(state_data: str):
+        try:
+            return json.loads(state_data)
```

Comment:
non-blocking nit: might be nice to have this `from_json()` return an explicit dict - but i think it's more likely that we'll delete this than change the fields for this class, so this should be fine as-is/we can make this look like the others later if this somehow still sticks around once the batch boxes go away and we get rid of the ssh executor code :p